set.mm

```diff
@@ -231,11 +231,12 @@ surrounded by whitespace (space, tab, CR, LF, or FF).
       file or has been already been inserted, it will not be inserted again.
 
 Inside of comments, it is recommended that labels be preceded with a tilde (~)
-and math symbol tokens be enclosed in grave accents (` `).  This way the LaTeX
-and HTML rendition of comments will be accurate, and (future) tools to globally
-change labels and math symbols will also change them in comments.  Note that
-inside of grave accents a pair of grave accents is interpreted as a single
-grave accent.  A special comment containing $ t (with no space after the dollar
+and math symbol tokens be enclosed in backticks, or grave accents (` `).  These
+tildes, tokens, math symbols and backticks should be surrounded by spaces.
```

Comment:
Until now, it is not forbidden/discouraged to surround backticks by spaces in comments: Only after the opening and before the closing backtick a space is needed. There is no need to have a space before the opening and after the closing backtick, see, for example, ``wff variable (usually ` ph `) or ...`` or ``...including that wff variable (` ph `).``

This topic was already discussed elsewhere...

Reply:
Not forbidden, but the rewrapping automatically adds spaces there, so better ask contributors to put spaces, to avoid CI test fails.

Reply:
> Not forbidden, but the rewrapping automatically adds spaces there, so better ask contributors to put spaces, to avoid CI test fails.

So for comments which are not rewrapped (section header comment, comments in HTML) spaces can be omitted.

Reply:
Yes, a loophole in the "rewrap" command allows contributors to use non-consistent spacing. Not sure this is worth mentioning in that non-displayed comment.

Reply:
> Yes, a loophole in the "rewrap" command allows contributors to use non-consistent spacing. 

Yes , the usage of spacing is not consistent in this case, but as far as I remember there was no agreement how it should be made consistenmt (to enforce a space before the opening and after the closing backtick or not in general).

> Not sure this is worth mentioning in that non-displayed comment.

OK, then I see no need to have changed this text at all.

But let's stop the discussion now. We can leave the text now as it is.